notebooks/RAG_evaluation.ipynb

```diff
@@ -0,0 +1,979 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.        N_GENERATIONS = 30
</pre><p>Why 30? How do we choose this number? How do we know if it's enough?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
This is completely arbitrary: for a real usecase, since ~half of the generated questions are excluded by the filters, I'd say we should generate 100 questions at least.
But for an explanatory notebook, for the sake of speed & cost for the user, maybe we should generate only 10 to show the process, then load the pregenerated dataset for the remaining questions, what do you think ?

Reply:
I think it's ok to generate only 10 in the notebook, but it would be super useful to add these considerations in the notebook.

Reply:
<p>Thank you! Adding this in the notebook.</p>